_data/azaya89/README.md

```diff
@@ -0,0 +1,62 @@
+# [Azaya](https://github.com/Azaya89)'s Zarr collection.
+
+## Twitter Mentions:
+
+
+- https://twitter.com/rajadain/status/1574097645806682121
+
+- https://twitter.com/latlong_blog/status/1543726888908009472?s=20&t=-ydMJPjiK47szWDjB7EIZw
+
+- https://twitter.com/ruth_mottram/status/1580130892692918272?s=20&t=-ydMJPjiK47szWDjB7EIZw
+
+-----
+
+## Youtube mentions:
+
+
+- https://www.youtube.com/watch?v=RLHM5MQ5kAs
+
+- https://www.youtube.com/watch?v=0bqpxX3Nn_A
+
+- https://www.youtube.com/watch?v=oWltJsPlIaQ&t=173s
+
+----
+
+
+## Code Snippets and visualisation:
+
+
+- [Interactive design of GPU-accelerated Image Data Flow Graphs and cross-platform deployment using multi-lingual code generation](https://haesleinhuepf.github.io/BioImageAnalysisNotebooks/32_tiled_image_processing/tiled_image_file_formats_zarr.html)
```

Comment:
The title and the link to the code don't match. Can you please correct the title?

Reply:
I have resolved it now. Thank you for pointing it out.